### DISCUSSION TITLE
Version of Birt to use with Java 11

### DISCUSSION THREAD

**Discussion by jmenzies12** [eclipse-birt/birt#752] [Q&A] [answered]
Hey there,

We are currently using Birt release 4.6.0 with Java 8 and are planning on upgrading to Java 11 in the coming months. Initially we thought Birt was no longer being worked on but I now see that there is a version 4.9.0 in the works, really glad this is happening and appreciate everyone working on it.

4.9.0 looks to support java 11 when it's released but are there any older releases that already support it ? I cannot find any releases after 4.6.0.

Cheers!
Jonathan

**Comment by wimjongman** [comment, marked as answer]
I think you can run BIRT 4.6 onwards on java 11. Why would that not work?

The latest version is 4.8.0 but you can already download stable version 4.9.0 please take a look at the [README](https://github.com/eclipse/birt/blob/master/README.md)

**Reply by colinsutton** [reply]
I am running BIRT 4.8 on windows 10 with Java 11.0.12, building and testing reports with no issues.
